### ISSUE TITLE
Name of app differs in dev versus production mode

### ISSUE THREAD

**Issue by rickhall** [SimulatedGREG/electron-vue#118]
When I run my app with "npm run dev" and execute the following code:

    console.log(this.$electron.remote.app.getPath("userData"));

I get the value "/Users/rhall/Library/Application Support/Electron". However, when I run the production app, I get "/Users/rhall/Library/Application Support/MyApp".

I would like to use the userData directory based on my application name in both modes, is there a simple way to do this? Thanks.

**Comment by SimulatedGREG** [comment]
@rickhall 

There really isn't a way to keep that constant between development and production. This has to do with how `electron-prebuilt` is used in development and the production app uses a fresh electron setup.

**Comment by rickhall** [comment]
That is a bummer.

If you are forced to have "Electron" be your app name during development, how do you handle developing multiple apps at the same time if they are all clashing in the the same userData directory? Just curious.

**Comment by SimulatedGREG** [comment]
You could namespace your data during development (or both, since it'll work either way), like putting all assets in a specific folder or appending a file pre-fix for all files.

**Comment by rickhall** [comment]
Ok, not great, but do-able. Thanks for your quick responses.

**Comment by citycide** [comment]
@rickhall I ran into this also, and I think I had it worked out. You need to make sure you have a `productName` field in your `package.json` because Electron will use that as your app name. Pretty sure I also used [`app.setName()`](https://github.com/electron/electron/blob/master/docs/api/app.md#appsetnamename) when in dev mode. If you really want to be sure, you could probably use [`app.setPath()`](https://github.com/electron/electron/blob/master/docs/api/app.md#appsetpathname-path).

**Comment by rickhall** [comment]
@citycide Thanks for the tips.

I tried setting productName (in either and both package.json files) and it seemed to have no effect. Too bad, because that would have been perfect. I also couldn't seem to get app.setName() to create the desired effect (perhaps I wasn't doing it at the right time).

However, I did get app.setPath() to give me the desired effect. I simply constructed and set the userData path in the end of the index.dev.js. Would be better if I could just modify package.json, but this is better than nothing. Thanks!
